src/main/java/io/zeebe/monitor/config/KafkaConfig.java

```diff
@@ -27,6 +27,10 @@ public class KafkaConfig {
     public int numOfThreads;
     @Value("${kafka.streams.maxMessageSizeBytes:5242880}")
     private int maxMessageSizeBytes;
+    @Value("${kafka.streams.maxPollRecords:30}")
+    public int maxPollRecords;
+    @Value("${kafka.streams.maxPollIntervalMs:600000}")
+    public int maxPollIntervalMs;
```

Comment:
Should these defaults be the currently used values?

Reply:
No,
this is intended value. if it works fine, we don't have to change anything on chart